src/image/Calib.cc

```diff
@@ -392,31 +393,41 @@ std::pair<ndarray::Array<double,1>, ndarray::Array<double,1> > Calib::getMagnitu
 
 namespace {
 
-class CalibSchema {
+int const TableVersion = 2;  // current version of ExposureTable
+std::string const ExpTimeFieldName = "exptime";  // name of exposure time field
```

Comment:
I can't see having constants that start with lowercase so I'll put up with UPPER_CASE